pkg/cmd/admin/policy/policy.go

```diff
@@ -64,8 +64,9 @@ func NewCmdPolicy(name, fullName string, f *clientcmd.Factory, out, errout io.Wr
 			},
 		},
 		{
-			Message: "Assign cluster roles to users and groups:",
+			Message: "Create/assign cluster roles to users and groups:",
 			Commands: []*cobra.Command{
+				NewCmdCreateClusterRole(CreateClusterRoleRecommendedName, fullName+" "+CreateClusterRoleRecommendedName, f, out),
```

Comment:
> I don't see "create" subtree in oadm command. I see only commands with prefix "create-" in the "Configuration" subtree. Did you mean this section?

`oc create clusterrole` is appropriate since its an API resource.